DataFormats/Detectors/PHOS/include/DataFormatsPHOS/Cluster.h

```diff
@@ -0,0 +1,88 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_PHOS_CLUSTER_H_
+#define ALICEO2_PHOS_CLUSTER_H_
+
+#include "DataFormatsPHOS/Digit.h"
+
+namespace o2
+{
+namespace phos
+{
+class Geometry;
+/// \class Cluster
+/// \brief Contains PHOS cluster parameters
+
+class Cluster
+{
+
+ public:
+  Cluster() = default;
+  Cluster(const Cluster& clu);
```

Comment:
User-defined copy constructor makes class non-POD (though there are tricks to declare it as POD) but your copy constructor makes the same what 
``Cluster(const Cluster& clu) = default`` would do, please remove it.